helix-term/src/commands/engine/steel.rs

```diff
@@ -693,6 +709,14 @@ fn load_configuration_api(engine: &mut Engine, generate_sources: bool) {
 "#,
         ));
 
+        builtin_configuration_module.push_str(&format!(
+            r#"
+(provide get-config-option-value)
+(define (get-config-option-value arg)
+    (helix.get-config-option-value *helix.cx*))
+"#,
+        ));
+
```

Comment:
Hi @mattwparas, thanks for your work on this PR! 🎉 

I noticed that in the call to `helix.get-config-option-value` in the code-gen, the argument `arg` is unused. From looking at the surrounding context, it seems like this argument is necessary to ensure being passed to the `helix.get-config-option-value`. 

Would it make sense to pass the argument like this: `(helix.get-config-option-value *helix.cx* arg)`? Let me know if I can help further with this. Thanks again for the great work!